components/List.js

```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components";
+import React from "react";
+
+export default function List(props) {
+  return (
+    <>
+      {props.items.map((item) => (
+        <li key={item.id}>
+          <ItemLayout>
+            <TitleText>{item.title}</TitleText>
+
+            <AmountText
+              style={{
+                color:
+                  item.amount > 0
+                    ? "var(--positive-balance)"
+                    : "var(--negative-balance)",
+              }}
+            >
+              {item.amount} $
+            </AmountText>
+          </ItemLayout>
+        </li>
+      ))}
+    </>
+  );
+}
+
+const ListLayout = styled.div`
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1em;
+  margin-left: 2em;
+  margin-right: 2em;
+`;
+
+const ItemLayout = styled.div`
```

Comment:
Same here, you could use 
`const ItemLayout = styled.li``
`
and drop the unnecessary div